vm/systemSmartContracts/delegation.go

```diff
@@ -1087,12 +1121,6 @@ func (d *delegation) delegate(args *vmcommon.ContractCallInput) vmcommon.ReturnC
 	return d.delegateUser(args.CallValue, args.CallerAddr, args.RecipientAddr, dStatus)
 }
 
-func (d *delegation) addNewDelegatorToList(dStatus *DelegationContractStatus, address []byte) error {
```

Comment:
Those changes are incompatible with what we have on the testnet right now. Will provide a new, fresh testnet in order to test these in the wild.

Reply:
yep. this list would have limited us to 22K users per delegation contract.